### PR TITLE
Table refactor

### PR DESCRIPTION
**Pull request by GustavoZHA**

### CODE REVIEW

File: src/main/java/salesforce/config/users/User.java [hunk 1]
```diff
@@ -47,4 +48,18 @@ public String getPassword() {
     public void setPassword(String password) {
         this.password = password;
     }
+
+    /**
+     * Sets the user's alias.
```
Comment (JorgeRCV) [inline src/main/java/salesforce/config/users/User.java]: Gets instead of sets

File: src/main/java/salesforce/config/users/User.java [hunk 2]
```diff
@@ -47,4 +48,18 @@ public String getPassword() {
...
+
+    /**
+     * Sets the user's alias.
+     */
+    public String getAlias() {
+        return alias;
+    }
+
+    /**
+     * Gets the user's alias.
```
Comment (JorgeRCV) [inline src/main/java/salesforce/config/users/User.java]: Sets

File: src/main/java/salesforce/ui/pages/tables/TablesValuesDefect.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                return individualTableValue(tableFeature);
+            case "campaign":
+                return campaignTableValue(tableFeature);
+            default:
+                throw new InvalidArgumentException("Invalid feature or add this feature");
+        }
+    }
+
+    /**
+     * Sets some default or autogenerate value of work type.
```
Comment (JorgeRCV) [inline src/main/java/salesforce/ui/pages/tables/TablesValuesDefect.java]: It should be gets

File: src/main/java/salesforce/ui/pages/tables/TablesValuesDefect.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+     *
+     * @param tableFeature is some values of table
+     * @return a list with values of table
+     */
+    private List<String> workTypeTableValue(Map<String, String> tableFeature) {
+        return new ArrayList<String>(tableFeature.values());
+    }
+
+    /**
+     * Sets some default or autogenerate value of contract.
```
Comment (JorgeRCV) [inline src/main/java/salesforce/ui/pages/tables/TablesValuesDefect.java]: All java doc in this class says sets but are getters

File: src/test/java/org/salesforce/hooks/GeneralHooks.java [hunk 5]
```diff
@@ -21,11 +21,10 @@
     private WebDriverManager webDriverManager;
     private PersonalInformation personalInformation;
 
-    public GeneralHooks(final WebDriverManager webDriverManager, final PersonalInformation personalInformation) {
```
Comment (JorgeRCV) [inline src/test/java/org/salesforce/hooks/GeneralHooks.java:24]: Checkstyle test does not pass in this class

File: src/test/java/org/salesforce/steps/featureSteps.java [hunk 6]
```diff
@@ -5,9 +5,11 @@
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+
```
Comment (JorgeRCV) [inline src/test/java/org/salesforce/steps/featureSteps.java]: You didnt ordered the imports according to the checkstyle and team conventions

**Review by JorgeRCV — APPROVED** [review]

**Review by JorgeRCV — COMMENTED** [review]
Use check style for the test please

**Review by cristianfchq — APPROVED** [review]

File: src/test/java/org/salesforce/steps/featureSteps.java (hunk 6 above)
Comment (GustavoZHA) [reply src/test/java/org/salesforce/steps/featureSteps.java]: Fix this problem is part of other member of team

**Pull request merged**